pepper-apis/studybuilder-cli/studies/atcp/i18n/pt.json

```diff
@@ -80,6 +80,7 @@
   "consent_contact_detail_br4": "",
   "consent_contact_detail_br5": "",
   "consent_contact_detail_email_help": "",
+  "consent_contact_detail_actual_email": "<a class=\"Link\" href=\"mailto:Help@wirb.com\">Help@wirb.com</a>",
```

Comment:
consent_contact_detail_br1 is missing the address.  Expected?

Reply:
For some reason translation didn't exist for this language. 
Patch isn't updating this language . Added a note to check when we add translations